js/main.js

```diff
@@ -0,0 +1,27 @@
+$(document).ready(function () {
+  var myInit = { method: 'GET',
+               cache: 'default' };
+
+  fetch('https://cors-anywhere.herokuapp.com/https://api.meetup.com/slcdevs/events?&amp;sign=true&amp;photo-host=public&amp;page=20', myInit)
+    .then(function (res) {
+      return res.json()
+    })
+    .then(function (myJson) {
+      let title = myJson[0].name;
+      let description = myJson[0].description;
+      let link = myJson[0].link;
+      let address = myJson[0].venue.address_1;
```

Comment:
Double check that `venue` isn't null, as not all future events have their venue set